tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -377,6 +379,7 @@ def test_merge_with_already_existing_article_in_the_db(
     assert obj.status == ObjectStatus.COMPLETED
     assert obj.extra_data['is-update'] is True
     assert obj.extra_data['merged'] is True
+    assert are_files_consistent(obj.data['control_number'])
```

Comment:
imo this is not the nicest way of checking this, as what you'll get in the case of failure is `assert False`, and you will not really have the actual stuff it was really checking. I would have created a function called `assert_files_consistent` that instead of returning `True` or `False` actually does an `assert` inside, that way if it fails, you end up in that namespace with access to the actual variables that were being checked, that helps a lot debugging. Probably to keep in mind in the next refactor.